python/lsst/cp/pipe/ptc/cpSolvePtcTask.py

```diff
@@ -463,6 +466,17 @@ def run(self, inputCovariances, camera=None, detId=0):
             # PhotonTransferCurveDataset object.
             datasetPtc = self.fitMeasurementsToModel(datasetPtc)
 
+        # Initial validation of PTC fit.
+        for ampName in ampNames:
+            noise = np.nanmedian(datasetPtc.noiseList[ampName])**2
+            noiseFitted = datasetPtc.noise[ampName]
+
+            # Check if noise is close to noiseFitted
+            if not np.isclose(noiseFitted, noise, rtol=0.05, atol=0.0):
```

Comment:
I think we want this comparison in ADU (dispersion-space) not ADU^2 (variance-space).  That matches the comment text better, and corresponds to how we use "read noise" elsewhere.

Reply:
Good suggestion